file-formats/ront/type/zif_aff_ront_v1.intf.abap

```diff
@@ -0,0 +1,71 @@
+INTERFACE zif_aff_ront_v1
+  PUBLIC.
+
+  CONSTANTS:
+    BEGIN OF co_category,
+      "! <p class="shorttext">Business Object</p>
+      "! Business object
+      business_object      TYPE c LENGTH 2 VALUE `BO`,
+      "! <p class="shorttext">Technical Object</p>
+      "! Technical object
+      technical_object     TYPE c LENGTH 2 VALUE `TO`,
+      "! <p class="shorttext">Analytical Object</p>
+      "! Analytical object
+      analytical_object    TYPE c LENGTH 2 VALUE `AO`,
+      "! <p class="shorttext">Configuration Object</p>
+      "! Configuration object
+      configuration_object TYPE c LENGTH 2 VALUE `CO`,
+    END OF co_category.
+
+  TYPES:
+    "! <p class="shorttext">Node</p>
+    BEGIN OF ty_rontnode,
+      "! <p class="shorttext">Name</p>
+      "! Node name
+      "! $required
+      node_name     TYPE c LENGTH 40,
+
+      "! <p class="shorttext">Root Node Flag</p>
+      "! Root node flag
+      "! $required
```

Comment:
Is this field really required?
If we remove the required tag then "isRoot": false, is not printed into the json only "isRoot": true is printed to the json. As I understood in the node_types table "isRoot" is always false and is also not editable to true, so we can ommit "isRoot" (atleast in the node_types table). I would remove the required tag

Reply:
Do we even need the "isRoot" information or can we remove it?

Is it working like this?

![image](https://user-images.githubusercontent.com/80829269/163954630-07f2b568-b045-45b3-824d-b66f17197d69.png)

Reply:
I would have the same question. Since we distinguish between root node types and node types, do you still need the field `is_root`?

Reply:
Yes, you are right. By distinguishing between the root SAP Object Node Type in component root_node_type and the other SAP Object Node Types in table nodeTypes the information about which node type "is root" is already encoded. This makes the component is_root unnecessary. 
Thanks for making this good point!